openassessment/data.py

```diff
@@ -1577,3 +1565,125 @@ def get_file_uploads(self, missing_blank=False):
                 files.append(file_upload)
             self.file_uploads = files
         return self.file_uploads
+
+
+def score_type_to_string(score_type):
+    """
+    Converts the given score type into its string representation.
+    """
+    SCORE_TYPE_MAP = {
+        PEER_TYPE: "Peer",
+        SELF_TYPE: "Self",
+        STAFF_TYPE: "Staff",
+        }
+    return SCORE_TYPE_MAP.get(score_type, "Unknown")
+
+def parts_summary(assessment_obj):
+    """
+    Retrieves a summary of the parts from a given assessment object.
+    """
+    return [
+        {
+            'criterion_name': part.criterion.name,
+            'score_earned': part.points_earned,
+            'score_type': part.option.name if part.option else "None",
+        }
+        for part in assessment_obj.parts.all()
+    ]
+
+def get_scorer_data(anonymous_scorer_id, user_data_mapping):
+    """
+    Retrieves the scorer's data (full name, username, and email) based on their anonymous ID.
+    """
+    scorer_data = user_data_mapping.get(anonymous_scorer_id, {})
+    scorer_name = scorer_data.get('fullname', "Unknown")
+    scorer_username = scorer_data.get('username', "Unknown")
+    scorer_email = scorer_data.get('email', "Unknown")
+    return scorer_name, scorer_username, scorer_email
```

Comment:
```suggestion
    return (
         scorer_data.get('fullname', "Unknown"),
         scorer_data.get('username', "Unknown")
         scorer_data.get('email', "Unknown")
     )
```
can we change the unknown for an empty string?